### PR TITLE
Upgrade to Angular 13

### PR DESCRIPTION
**Pull request by jlp0328**

### CODE REVIEW

**Review by julienCarret — APPROVED** [review]

**Review by julienCarret — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# 1.5.3 (2022-02-16)
```
Comment (mpellerin42) [inline CHANGELOG.md]: wrong version

**Review by mpellerin42 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (jlp0328) [reply CHANGELOG.md]: Thank you!

**Review by mpellerin42 — APPROVED** [review]

**Review by ebrehault — COMMENTED** [review]
Hi @jlp0328 !
The CI build fails when trying to build the lib.
Try to run `npm run build:lib` locally to see what's wrong.
If you cannot fix it easily, ping me, I'll have a look.
And if you are in a hurry, you can just use `branch: dev-angular13-upgrade` in your `mrs.developer.json` instead of a tag, so you can build your own project immediately.

**Review by ebrehault — APPROVED** [review]

**Comment by ebrehault** [comment]
Great! Thanks!

**Comment by jlp0328** [comment]
Thank you @ebrehault -- I had to manually update everything in package.json and missed ng-packagr 🙄 😄 

I appreciate you swooping in!! I hope you're doing well!! We miss you!! ❤️

**Pull request merged**